### DISCUSSION TITLE
Integration with QuestDB

### DISCUSSION THREAD

**Discussion by HacktopusX** [pg-sharding/spqr#798] [Ideas]
QuestDB is a timeseries database that supports Postgres wire protocol 
https://questdb.io/docs/reference/api/postgres/

But Quest dose not support sharding on open source version (HA is avaiable on enterprise)
Has anyone tried to use PGWire feature of QuestDB with SPQR to have HA and horizontal scaling on QuestDB ?

**Comment by x4m** [comment]
SPQR requires Postgres protocol + some subset of Postgres SQL DML + some catalog informaion.
If in QuestDB tables are selected\updated in a same fashion as in vanilla Postgres - everything should just work.
